templates/_helpers.tpl

```diff
@@ -140,7 +140,7 @@ Set's additional environment variables based on the mode.
 {{- define "vault.envs" -}}
   {{ if eq .mode "dev" }}
             - name: VAULT_DEV_ROOT_TOKEN_ID
-              value: "root"
+              value: {{ .Values.server.dev.devRootToken | default "root" }}
```

Comment:
Instead of having `default "root"` here, set this in the `values.yaml` file and have it uncommented. We've been moving away from backend defaults to be more transparent in the `values.yaml` file.